examples/ArduinoIoTCloud_Travis_CI/thingProperties.h

```diff
@@ -8,8 +8,9 @@
 #if defined(BOARD_HAS_WIFI)
 #elif defined(BOARD_HAS_GSM)
 #elif defined(BOARD_HAS_LORA)
+#elif defined(BOARD_HAS_NB)
 #else
-  #error "Arduino IoT Cloud currently only supports MKR1000, MKR WiFi 1010, MKR GSM 1400 and MKR WAN 1300/1310"
```

Comment:
Was it intentional to remove MKR WAN 1300/1310 from the error message?

Reply:
They were not removed but moved towards the end ;)

Reply:
I still don't see it:
https://github.com/arduino-libraries/ArduinoIoTCloud/blob/138f726a30e042d2b41df764e7e97ff240099eb3/examples/ArduinoIoTCloud_Travis_CI/thingProperties.h#L13
but I notice now this is just the CI sketch, so not really important either way.

Reply:
Oh my, you are right. Fixed it. Interestingly enough I clearly remember preserving those changes ...